src/task1_1.js

```diff
@@ -0,0 +1,3 @@
+process.stdin.on('data', data => {
+  console.log([...data.toString().trim()].reverse().join('') + '\n');
```

Comment:
You should put the result into write stream instead of console log

Reply:
fixed